build/scripts/build.ps1

```diff
@@ -628,7 +663,7 @@ try {
         $bootstrapDir = Make-BootstrapBuild
     }
 
-    if ($build -or $pack) {
+    if ($build -or $buildCoreClr -or $pack) {
```

Comment:
I think we should be consistent with `$buildAll` here. Inside `Process-Arguments` we should set `$build` to `$true` if `$buildCoreClr` is also `$true`.